### DISCUSSION TITLE
Confirmation on the eligibility of non open sourced code that exists in public repos

### DISCUSSION THREAD

**Discussion by andrewfenn** [community/community#23313] [New to GitHub]
<p>On the <a href="https://github.com/pricing" rel="nofollow noopener">pricing page</a> you list a couple of items which are free for public repos under team and enterprise editions. Do those same free items for public repos extend to public repos with “non-free” (i.e code not under an open source licence) code in them or does “free for public repositories” come under the unstanding that the code in these repos needs to be under and open source licence; or be open source in some way? Could I for example publish a commercial venture under a public repo (understanding that the code would be viewable but still copyrighted only to me) and still gain all the same benefits an open source project would or would this be shutdown by Github staff?</p>

**Comment by wordsoup** [comment, marked as answer]
<p>Hi <a class="mention" href="/andrewfenn">@andrewfenn</a>,</p>
<p>Not to worry! You’re free to choose whichever license best fits your use case, as long as you’re comfortable with these repositories being public. We have a complete breakdown of our license options <a href="https://help.github.com/en/github/creating-cloning-and-archiving-repositories/licensing-a-repository#choosing-the-right-license" title="GitHub Help: License Options" rel="nofollow noopener">here</a>, so please choose the license that best matches your needs.</p>
